CODEOWNERS

```diff
@@ -228,6 +228,7 @@ homeassistant/components/homeassistant/* @home-assistant/core
 homeassistant/components/homekit/* @bdraco
 homeassistant/components/homekit_controller/* @Jc2k @bdraco
 homeassistant/components/homematic/* @pvizeli @danielperna84
+homeassistant/components/homewizard_energy/* @DCSBL
```

Comment:
This integration should be called `homewizard` as that's the brand name.

Reply:
HomeWizard is the company and it also has an (obsolete) product called 'HomeWizard'. This integration is developed for a new product category 'HomeWizard Energy' and is not developed with the intention to support other products than the products with this name.

IMO, it is confusing to have the name `homewizard` as it implies that is designed for the old 'HomeWizard'. And 'energy` is to generic (and also used of course). What do you think?

Reply:
Let's stick with the company name. We can just document it's for energy devices only.

Reply:
Changed domain to `homewizard` (https://github.com/home-assistant/core/pull/55812/commits/54648e39477377658d8a659b8de6b8e80ca5dad3). I think we should update some metadata for PR now, what is the best approach?
 
The brand should be added/changed to. Do we stick with the HomeWizard Energy logo (Like 'the Philips Hue' logo) or do we use the HomeWizard logo (like a 'Philips' logo)